### DISCUSSION TITLE
Add option to autoconnect/disconnect from chat when stream goes on/off

### DISCUSSION THREAD

**Discussion by zalupax** [Chatterino/chatterino2#3417] [Ideas] [answered]
If its possible to add an options to automatic connect and disconnect when stream change his online status? This will prevent lags when using multiple accounts and avoid bans on some streamers.

**Comment by Felanbird** [comment, marked as answer]
> and avoid bans on some streamers.

No

**Reply by zalupax** [reply]
Why not? Some moders just banned for being in chat for too long when stream goes off..

**Reply by Mm2PL** [reply]
That's a you problem not a problem of the chat client. I've never seen this kind of behavior before.

**Reply by zalupax** [reply]
> That's a you problem not a problem of the chat client. I've never seen this kind of behavior before.

if you have not seen this does not mean that this problem does not exist

**Reply by Felanbird** [reply]
> if you have not seen this does not mean that this problem does not exist

That is not a problem, it's called insanity. You want us to intentionally add a feature that directly bypasses an intended feature implemented by Twitch, because... mods ban you when the stream is over....???

If this is the case, there's 3 possibilities:
1. You are intentionally ban evading.
2. The mods of that stream are intentionally banning people to prevent the streamer from having viewers and should be unmodded.
3. The streamer doesn't care about his viewers and wants them banned for some reason.

No matter the reason, we are not going to circumvent Twitch features and risk Twitch banning Chatterino from connecting to their servers. Watch somebody who at least pretends to care about their viewers instead.
